docs/Migrating.md

```diff
@@ -14,7 +14,7 @@
 
 * Discrete Actions now have branches. You can now specify concurrent discrete
```

Comment:
I know this is not part of the PR - but we ought to better explain `branches` or at a minimum add a reference.